server/container/guice/protocols/webadmin-mailbox/src/main/java/org/apache/james/modules/server/MessagesRoutesModule.java

```diff
@@ -30,5 +30,7 @@
     protected void configure() {
         Multibinder<Routes> routesMultibinder = Multibinder.newSetBinder(binder(), Routes.class);
         routesMultibinder.addBinding().to(MessagesRoutes.class);
+
+
```

Comment:
You miss this line:

```
Multibinder.newSetBinder(binder(),
                TaskFromRequestRegistry.TaskRegistration.class, Names.named(MessagesRoutes.ALL_MESSAGES_TASKS));
```

Declare an empty multibinder if none provided.

Ill be essential for memory that have /messages but probably no tasks for it...